tinygrad/runtime/ops_metal.py

```diff
@@ -71,13 +72,23 @@ def as_buffer(self, src:Any) -> memoryview:
     return src.contents().as_buffer(src.length())
   def copyin(self, dest:Any, src:memoryview): self.as_buffer(dest)[:] = src
   def copyout(self, dest:memoryview, src:Any): dest[:] = self.as_buffer(src)
+  def load_buffer(self, dest:Any, src:Any, handles={}):
+    if (path := src.device.split(":")[1]) not in handles:
+      handles[path], err = self.device.device.newIOHandleWithURL_error_(NSURL.fileURLWithPath_(path), None)
+      assert err is None, "failed to load from disk"
+    cbuf = self.device.mtl_io_queue.commandBuffer()
+    cbuf.loadBuffer_offset_size_sourceHandle_sourceHandleOffset_(dest._buf, 0, dest.size*dest.dtype.itemsize, handles[path], src._buf.offset)
```

Comment:
The wrong type is being passed in here

Reply:
dest._buf - metal buffer
0 - int
dest.size*dest.dtype.itemsize - int
handles[path] - mtliofilehandle
src._buf.offset - int

I guess everything is right